### DISCUSSION TITLE
Version 2.7.8 latest stable or beta version.

### DISCUSSION THREAD

**Discussion by degrashopper** [johanmeijer/grott#288] [General]
Hi I'm a bit confusted.
Is latest released version 2.7.8 now the latest stable version or beta version.
The latest version on hub.docker is an older version then 2.7.8.

**Comment by johanmeijer** [comment]
Yes. 2.7.8 is latest stable version. I have had time yet to change the docker for that.

2.7.8 is available at docker but not with the latest tag. I will change that soon.

**Reply by degrashopper** [reply]
Okay Great! thanks for your quick response. i will give tag 2.7.8 a try then.
